DocSum/docker/ui/react/README.md

```diff
@@ -0,0 +1,29 @@
+<h1 align="center" id="title">Doc Summary React</h1>
+
+### 📸 Project Screenshots
+
+![project-screenshot](../../../assets/img/docsum-ui-react.png)
+![project-screenshot](../../../assets/img/docsum-ui-react-file.png)
+![project-screenshot](../../../assets/img/docsum-ui-react-error.png)
+
+<h2>🧐 Features</h2>
+
+Here're some of the project's features:
+
+- Summarizing Uploaded Files: Upload files from their local device, then click 'Generate Summary' to summarize the content of the uploaded file. The summary will be displayed on the 'Summary' box.
+- Summarizing Text via Pasting: Paste the text to be summarized into the text box, then click 'Generate Summary' to produce a condensed summary of the content, which will be displayed in the 'Summary' box on the right.
+- Scroll to Bottom: The summarized content will automatically scroll to the bottom.
+
+<h2>🛠️ Get it Running:</h2>
+
+1. Clone the repo.
+
+2. cd command to the current folder.
+
+3. Modify the required .env variables.
+   ```
+   VITE_DOC_SUM_URL = ''
```

Comment:
The variable in your .env file is `VITE_BACKEND_SERVICE_ENDPOINT`, so why is it `VITE_DOC_SUM_URL` here?

Reply:
to run it locally it has to be VITE_DOC_SUM_URL